WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderNavigationMenu.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct ReaderNavigationMenu: View {
+
+    var viewModel: ReaderTabViewModel
+    var selectedItem: ReaderTabItem
+
+    var body: some View {
+        HStack {
```

Comment:
I think this needs a little bit of padding, but perhaps @osullivanchris can confirm further. In landscape orientation, specifically, the button felt too close to the top edge of the screen. When you scroll over an image with contrasting color, there appears to be an uneven padding gap between the top and bottom side of the new navigation button:

![simulator_screenshot_FF8587FF-9F00-482D-8636-13CF6CCD2110](https://github.com/wordpress-mobile/WordPress-iOS/assets/1299411/e07277b0-a3d1-4ab1-8a8f-13818a1751db)

Reply:
I agree that its too close to the top of the screen. and might be slightly so (by like 4px) in portrait too.

Reply:
I added 4px to the top of the view in https://github.com/wordpress-mobile/WordPress-iOS/pull/22260/commits/2535d17318e997dcc052780a117fdbdc3edaae34. Here's a screenshot:

![Simulator Screenshot - iPhone SE (3rd generation) - 2023-12-20 at 17 44 45](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/44c01b36-9580-4f88-ab98-50916ff8ac22)